### PR TITLE
PicoRV32 tutorial

### PR DESCRIPTION
**Pull request by WRansohoff**
We can split #1330 into this, and a future PR to deduplicate the quickstart and remote processing.

### CODE REVIEW

File: docs/tutorials/picorv32_ram.rst [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+    chip.load_target('skywater130_demo')
+    chip.input('picorv32.v')
+    chip.set('option', 'remote', True)
+    chip.run()
+
+If you run that example as a Python script, it should take approximately 20 minutes to run if the servers are not too busy. We have not added a RAM macro yet, but this script will build the CPU core with I/O signals placed pseudo-randomly around the edges of the die area. Once the job finishes, you should receive a screenshot of your final design, and a report containing metrics related to the build in ``build/picorv32/job0/report.pdf``. SiliconCompiler will try to open the file after the job completes, but it may not be able to do so if you are running in a headless environment.
+
+.. image:: ../_images/picorv32_report.png
+
+For the full GDS-II results and intermediate build artifacts, you can install the EDA tools on your local system, and run the same Python build script with the :keypath:`option, remote` parameter set to ``False``.
```
Comment (aulihant) [inline docs/tutorials/picorv32_ram.rst:41]: > For the full GDS-II results and intermediate build artifacts, you can follow the instructions for a :ref:`local run`.

Just point them to 3.2

File: docs/tutorials/picorv32_ram.rst [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+
+.. note::
+
+    We are not returning the full results during this early beta period because we want to minimize bandwidth, and we believe that our public beta is currently best suited for rapid prototyping and design exploration.
```
Comment (aulihant) [inline docs/tutorials/picorv32_ram.rst:45]: @gadfort and @WRansohoff this sound like we will be providing more results in a remote run later. 
What additional results will be provided later?

File: docs/tutorials/picorv32_ram.rst [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+
+We will use the `sky130_sram_2kbyte_1rw1r_32x512_8 <https://github.com/VLSIDA/sky130_sram_macros/tree/main/sky130_sram_2kbyte_1rw1r_32x512_8>`_ block in this example. You can download the required files through GitHub's website, or using a tool like  ``curl``::
+
+    curl https://raw.githubusercontent.com/VLSIDA/sky130_sram_macros/main/sky130_sram_2kbyte_1rw1r_32x512_8/sky130_sram_2kbyte_1rw1r_32x512_8.gds > sky130_sram_2kbyte_1rw1r_32x512_8.gds
+    curl https://raw.githubusercontent.com/VLSIDA/sky130_sram_macros/main/sky130_sram_2kbyte_1rw1r_32x512_8/sky130_sram_2kbyte_1rw1r_32x512_8.lef > sky130_sram_2kbyte_1rw1r_32x512_8.lef
+
+Once you have a GDS and LEF file for your RAM macro, create a new directory called ``sram/`` in same location as your PicoRV32 build files, and move the macro files there. Then, create a Python script called ``sky130_sram_2k.py`` in that ``sram/`` directory to describe the RAM macro in a format which can be imported by SiliconCompiler:
```
Comment (aulihant) [inline docs/tutorials/picorv32_ram.rst:61]: I would add this to be explicit about why we need those two files.

> **We will need a GDS and LEF file for the basic placement and routing information for this RAM macro.** Once you have those files, create a new directory called...

File: docs/tutorials/picorv32_ram.rst [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+    │ ├── sky130_sram_2k.bb.v
+    │ ├── sky130_sram_2k.py
+    │ ├── sky130_sram_2kbyte_1rw1r_32x512_8.gds
+    │ └── sky130_sram_2kbyte_1rw1r_32x512_8.lef
+    ├── picorv32.py
+    ├── picorv32.v
+    ├── picorv32_top.py
+    └── picorv32_top.v
+
+Your ``picorv32_top.py`` build script should take about 20 minutes to run on the cloud servers if they are not too busy, with most of that time spent in the routing task. As with the previous designs, you should see periodic updates on its progress, and you should receive a screenshot and metrics summary once the job is complete:
```
Comment (aulihant) [inline docs/tutorials/picorv32_ram.rst:237]: I would probably also replace "periodic" here with every 30 seconds to be explicit so as to set appropriate user expectations.

**Review by aulihant — CHANGES_REQUESTED** [review]
Overall looks good just some minor text clarity things.

File: docs/tutorials/picorv32_ram.rst (hunk 2 above)
Comment (gadfort) [reply docs/tutorials/picorv32_ram.rst:45]: maybe make the note: note we are only returning a limited amount of information (and point them to the reprocessing for more details)

**Review by gadfort — COMMENTED** [review]
Look good. I'll leave it to @aulihant to approve

**Review by aulihant — APPROVED** [review]
LGTM

**Pull request merged**